src/overlays/actors/ovl_En_Hg/z_en_hg.c

```diff
@@ -282,21 +298,21 @@ void func_80BCF95C(EnHg* this, PlayState* play) {
                     break;
                 case 2:
```

Comment:
```suggestion
                    break;
                    
                case 2:
```
Can you add some newlines between the breaks and cases